internal/cli/atlas.go

```diff
@@ -31,6 +31,8 @@ func AtlasBuilder() *cobra.Command {
 	cmd.AddCommand(AtlasBackupsBuilder())
 	cmd.AddCommand(AtlasEventsBuilder())
 	cmd.AddCommand(AtlasMeasurementsBuilder())
+	// cmd.AddCommand(AtlasLogsBuilder())
```

Comment:
[np] Can be removed now? Not blocking

Reply:
That's what I was working on and could not use as I needed the new format, I'll merge away as the next PR will remove the comments